README.md

```diff
@@ -35,9 +37,59 @@ USAGE:
     discord_bot start [OPTIONS]
 
 OPTIONS:
-    -g <GUILD_ID>        The guild ID (Server ID)
-    -h, --help           Print help information
-    -t <TOKEN>           The discord token
+    -g, --guild-id <GUILD_ID>
```

Comment:
Also idk if applicable here, but there should be a way to broadcast (/) global command to Discord 
See here : https://discord.com/developers/docs/interactions/application-commands#making-a-global-command